mappings/net/minecraft/advancement/criterion/EffectsChangedCriterion.mapping

```diff
@@ -16,6 +16,6 @@ CLASS net/minecraft/class_2027 net/minecraft/advancement/criterion/EffectsChange
 			ARG 0 source
 		METHOD method_8868 matches (Lnet/minecraft/class_3222;Lnet/minecraft/class_47;)Z
 			ARG 1 player
-			ARG 2 context
+			ARG 2 source
```

Comment:
This is a loot context. So
```suggestion
			ARG 2 context
```

Reply:
All the LootContexts in Criterion matches methods are named as if they aren't LootContexts.